solutions/simple/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0"
```

Comment:
you should bump this to >= 1.9.0 and use cross variable validation. There are many combinations / permutations so we badly need to use that kind of validation in the variables.

Reply:
done